egs/librispeech/asr/simple_v1/mmi_att_transformer_train.py

```diff
@@ -585,7 +608,7 @@ def main():
         cuts_dev,
         max_duration=max_duration,
```

Comment:
You have two options regarding the validation dataset:
1) distribute it to all nodes and synchronize dev loss
2) give 100% to all nodes and forego synchronization

By default, Lhotse's samplers are auto-distributing themselves when they detect distributed training. That means currently you are reporting the dev loss on a partial dev set. You need to either sync the loss across nodes or explicitly set rank=0 and world_size=1 to the dev sampler (to disable the auto-distribution mechanism).

It might be a good idea to explicitly set the rank and world size in train samplers too?

Also I'm not sure if this is a good (as in: least surprising) behaviour by Lhotse classes, I'm open to feedback.